blocks/library/text/index.js

```diff
@@ -47,7 +47,25 @@ registerBlock( 'core/text', {
 		}
 	],
 
-	edit( { attributes, setAttributes, insertBlockAfter, focus, setFocus } ) {
+	formatting: [
+		{
+			icon: 'editor-bold',
+			title: wp.i18n.__( 'Bold' ),
+			format: 'bold'
+		},
+		{
+			icon: 'editor-italic',
+			title: wp.i18n.__( 'Italic' ),
+			format: 'italic'
+		},
+		{
+			icon: 'editor-strikethrough',
+			title: wp.i18n.__( 'Strikethrough' ),
+			format: 'strikethrough'
+		}
+	],
+
+	edit( { attributes, setAttributes, insertBlockAfter, focus, setFocus, onFormatChange, formats } ) {
```

Comment:
Not specific to this pull request, but I'm concerned about how many props an implementer is expected to be aware of just to use an `Editable` component. Previously I'd suggested considering [`context`](https://facebook.github.io/react/docs/context.html) as a transparent pass-through. I don't really know whether it's a perfect solution, but we'll need to think of something. Maybe also something in the global state instead of `VisualBlock`'s state.